Doc/library/logging.handlers.rst

```diff
@@ -1148,6 +1148,11 @@ possible, while any potentially slow operations (such as sending an email via
    .. versionchanged:: 3.5
       The ``respect_handler_level`` argument was added.
 
+   .. versionchanged:: next
+      :class:`QueueListener` can now be used as a context manager via
+      :keyword:`with`. When entering the context, the listener is started. When
+      exiting the context, the listener is stopped.
```

Comment:
We should mention that when entering the context, the listener is returned by `__enter__`. Some people might otherwise write `listener = ...; with listener: ...` instead of an `with QueueListener() as listener: ...`. This is a very minor detail and I don't know if we're consistent in other places.

Reply:
Added.